### PR TITLE
Apply php/doc-en@082ddc1

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

**Comment by devnexen** [comment]
Looks correct to me

File: reference/pdo/pdostatement/fetch.xml [hunk 1]
```diff
@@ -120,14 +120,14 @@
      <listitem>
       <para> 
        Pour un objet PDOStatement représentant un curseur scrollable pour
-       lequel le paramètre <literal>cursor_orientation</literal> est défini
+       lequel le paramètre <literal>cursorOrientation</literal> est défini
```
Comment (Girgias) [inline reference/pdo/pdostatement/fetch.xml:123]:
```suggestion
       lequel le paramètre <parameter>cursorOrientation</parameter> est défini
```
Shouldn't the tag be changed here and below (or is this also an issue in EN?) ?

**Review by Girgias — APPROVED** [review]
LGTM other than the one question

File: reference/pdo/pdostatement/fetch.xml (hunk 1 above)
Comment (saundefined) [reply reference/pdo/pdostatement/fetch.xml]: Indeed, fixed

**Pull request merged**